sqlite3.go

```diff
@@ -119,3 +120,16 @@ func columnMapper(column *rel.Column) (string, int, int) {
 
 	return typ, m, n
 }
+
+func definitionFilter(table rel.Table, def rel.TableDefinition) bool {
+	_, ok := def.(rel.Key)
+	// https://www.sqlite.org/omitted.html
+	// > Only the RENAME TABLE, ADD COLUMN, RENAME COLUMN, and DROP COLUMN variants of the ALTER TABLE command are supported.
+	if ok && table.Op == rel.SchemaAlter {
```

Comment:
won't this skip `RENAME TABLE, ADD COLUMN, RENAME COLUMN, and DROP COLUMN` as well?

Reply:
oh, I missed `_, ok := def.(rel.Key)`

can you move above statement inside `if table.Op == rel.SchemaAlter`? so there's no need to try casting when Op is not SchemaAlter

Reply:
updated